cms/models.py

```diff
@@ -47,8 +81,94 @@ class HomePage(Page):
     ]
     parent_page_types = [Page]
     subpage_types = [
-        "CoursePage",
+        "CourseIndexPage",
         "ResourcePage",
+        "HomeProductPage",
+    ]
+
+    def _get_child_page_of_type(self, cls):
+        """Gets the first child page of the given type if it exists"""
+        child = self.get_children().type(cls).live().first()
+        return child.specific if child else None
+
+    @property
+    def product(self):
+        """
+        Gets the "HomeProductPage" section sub page.
+        """
+        return self._get_child_page_of_type(HomeProductPage)
+
+    def get_context(self, request, *args, **kwargs):
+        return {
+            **super().get_context(request),
+            "product_section_title": self.product.product_section_title
+            if self.product
+            else "Product Section Title",
+            "product_cards": self.product.product_cards if self.product else None,
+        }
+
+
+class HomeProductPage(Page):
```

Comment:
Not sure if you tried this already and it just didn't work out, but it makes much more sense and will be much more user friendly if this is not a child page of the home page, but a part of the home page itself. If we can find a way to have `product_section_title` as a field in `HomePage` and the `featured_products` inline panel also defined as a part of `HomePage`. Users are going to think of these elements as part of the home page and it would be best for them to see these inputs when they're editing the home page itself

Reply:
![screencapture-localhost-8013-cms-pages-6-edit-2021-08-06-13_00_53](https://user-images.githubusercontent.com/7334669/128509901-3669fafa-97ae-4811-9e00-e48e7c1fc801.png)

Reply:
@gsidebo is it good now ?